GPU/Vulkan/DebugVisVulkan.cpp

```diff
@@ -32,6 +32,8 @@
 #undef DrawText
 
 void DrawAllocatorVis(UIContext *ui, GPUInterface *gpu) {
+	/*
```

Comment:
Maybe we should just delete this for now and put a `// TODO`?  Git will remember.

-[Unknown]